packages/common/src/types/validation.ts

```diff
@@ -1,24 +1,72 @@
 import type { ChartKind } from './savedCharts';
 
-export type ValidationResponse = {
+type ValidationResponseBase = {
     validationId: number;
     createdAt: Date;
     name: string;
-    chartUuid?: string;
-    chartType?: ChartKind;
-    dashboardUuid?: string;
+    error: string;
+    errorType: ValidationErrorType | null;
```

Comment:
on validation response we can do `errorType?: validationErrorType` or `errorType: validationErrorType | undefined` 

afaik null is only used on db models

Reply:
amazing, thanks! yes yes